tools/nnicmd/launcher.py

```diff
@@ -53,8 +58,12 @@ def start_rest_server(port, platform, mode, experiment_id=None):
     stderr_full_path = os.path.join(HOME_DIR, str(port), 'stderr')
     stdout_file = open(stdout_full_path, 'a+')
     stderr_file = open(stderr_full_path, 'a+')
+    time_now = time.strftime('%Y-%m-%d %H:%M:%S',time.localtime(time.time()))
+    log_header = LOG_HEADER % str(time_now)
```

Comment:
Could you please add comment to explain what's the use of this log header format?